platform/android/java_class_wrapper.cpp

```diff
@@ -517,6 +517,19 @@ bool JavaClass::_call_method(JavaObject *p_instance, const StringName &p_method,
 		env->DeleteLocalRef(E);
 	}
 
+	jobject exception = env->ExceptionOccurred();
+	if (exception) {
+		env->ExceptionClear();
+
+		jclass java_class = env->GetObjectClass(exception);
+		Ref<JavaClass> java_class_wrapped = JavaClassWrapper::singleton->wrap_jclass(java_class);
+		env->DeleteLocalRef(java_class);
+
+		JavaClassWrapper::singleton->exception = Ref<JavaObject>(memnew(JavaObject(java_class_wrapped, exception)));
```

Comment:
```suggestion
		JavaClassWrapper::singleton->exception.instantiate(java_class_wrapped, exception);
```

Reply:
Ooo, thanks, I didn't even know that was a thing!